### PR TITLE
Fix Mode1UpLit causing iOS 9 / iOS 10 to break

### PR DESCRIPTION
**Pull request by iisa**

### CODE REVIEW

**Comment by cdrini** [comment]
Testing now :+1:

File: BookReaderDemo/demo-internetarchive.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
 
     <!-- JS dependencies -->
+    <script src="https://unpkg.com/@webcomponents/custom-elements"></script>
```
Comment (cdrini) [inline BookReaderDemo/demo-internetarchive.html:11]: @iisa @jbuckner Do we have a web components polyfill on IA? This was needed to get things working on iOS9.

Comment (jbuckner) [reply BookReaderDemo/demo-internetarchive.html:11]: Yeah we are using the `@webcomponents/webcomponentsjs` polyfill: https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs

Comment (cdrini) [reply BookReaderDemo/demo-internetarchive.html:11]: Sweet! I'll switch to that and test

Comment (cdrini) [reply BookReaderDemo/demo-internetarchive.html]: It didn't was because of this #477 , but the workaround was pretty simple.

**Comment by cdrini** [comment]
Testing parade:

- Mac (El Capitan) + Safari 9: 1up/2up display/work!
- Mac (Sierra) + Safari 10: 1up/2up display/work!
- Mac (High Sierra) + Safari 11: 1up/2up display/work!
- iOS 6S + Safari 9: 1up/2up display/work
    - A little janky (although hard to tell where it's coming from in Browserstack), but it works.
- iPad Air 2 + Safari 9: 1up/2up display/work!
    - The zoom/tripledot don't appear in top left for some reason
- iPad Pro + Safari 10: 1up/2up display/work!

File: src/BookNavigator/BookNavigator.js [hunk 2]
```diff
@@ -10,6 +11,8 @@ import BRFullscreenMgr from './br-fullscreen-mgr.js';
 import { Book } from './BookModel.js';
 import bookLoader from './assets/book-loader.js';
 
+const ResizeObserver = window.ResizeObserver || roPolyfill;
```
Comment (iisa) [inline src/BookNavigator/BookNavigator.js:14]: TY

**Review by iisa — COMMENTED** [review]
looks good. thanks for wrangling that ResizeObserver in there, too. heck of a find!

**Review by cdrini — APPROVED** [review]

**Comment by cdrini** [comment]
Thanks @iisa ! That was your PR! :) I just added the missing npm package. Should also help reduce the strain on our Sentry :P

**Pull request merged**